applications/structural_application/custom_python/add_custom_processes_to_python.cpp

```diff
@@ -49,16 +48,17 @@ namespace Kratos
 namespace Python
 {
 
-void AddCustomProcessesToPython()
+void AddCustomProcessesToPython(pybind11::module& m)
 {
-    using namespace boost::python;
+    using namespace pybind11;
```

Comment:
@vryy, we found that the "using pybind" to give problems in some contexts.

without the using it would be better, i would mean something like:

~~~cpp
namespace py = pybind11;

py::class_<TopologyUpdateProcess, Process >
    ("TopologyUpdateProcess", py::init<ModelPart&, double, double, double>())
    (m, "TopologyUpdateProcess")
    .def(init<ModelPart&, double, double, double>())
    .def("SetBinSize", &TopologyUpdateProcess::SetBinSize)
    .def("SetBinSize", &TopologyUpdateProcess::SetBinSize)
    .def("GetTopologyChange", &TopologyUpdateProcess::GetTopologyChange)
    .def("GetTopologyChange", &TopologyUpdateProcess::GetTopologyChange)
    .def("GetObjective", &TopologyUpdateProcess::GetObjective)
    .def("GetObjective", &TopologyUpdateProcess::GetObjective)
    .def(self_ns::str(self))
    // .def(self_ns::str(self))
    ;
    ;
~~~

@philbucher maybe you want to comment on this?

Reply:
`self_ns` does not exist in pybind, see [here](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/python/add_data_communicator_to_python.cpp#L137) how we do it now